### ISSUE TITLE
npm install failing on fresh clone

### ISSUE THREAD

**Issue by kav** [ericgio/react-bootstrap-typeahead#113]
clone a fresh copy of the repo, run npm install:
`npm install` fails on the call to `npm test` with

```
Chrome 54.0.2840 (Mac OS X 10.12.2) <TypeaheadMenu> renders a right-aligned typeahead menu FAILED
        AssertionError: expected 'dropdown-menu bootstrap-typeahead-menu dropdown-menu-justify' to include 'dropdown-menu-right'
            at Context.<anonymous> (webpack:///test/TypeaheadMenuSpec.js:41:34 <- tests.webpack.js:18941:47)
Chrome 54.0.2840 (Mac OS X 10.12.2) <TypeaheadMenu> renders a menu with a max-height of 200px FAILED
        AssertionError: expected '300px' to equal '200px'
            at Context.<anonymous> (webpack:///test/TypeaheadMenuSpec.js:46:40 <- tests.webpack.js:18946:53)
....
Chrome 54.0.2840 (Mac OS X 10.12.2) <TypeaheadMenu> does not show a paginator if `paginate=false` FAILED
        AssertionError: expected 1 to equal 0
            at Context.<anonymous> (webpack:///test/TypeaheadMenuSpec.js:108:37 <- tests.webpack.js:18991:50)
Chrome 54.0.2840 (Mac OS X 10.12.2) <TypeaheadMenu> displays custom pagination text FAILED
        AssertionError: expected 'Display additional results...' to equal 'See All'
            at Context.<anonymous> (webpack:///test/TypeaheadMenuSpec.js:121:50 <- tests.webpack.js:19001:63)
......
Chrome 54.0.2840 (Mac OS X 10.12.2) <Typeahead> should display a menu item for pagination FAILED
        AssertionError: expected 'Display additional results...' to equal 'See More'
            at Context.<anonymous> (webpack:///test/TypeaheadSpec.js:133:50 <- tests.webpack.js:19138:63)
Chrome 54.0.2840 (Mac OS X 10.12.2) <Typeahead> should not display a menu item for pagination FAILED
        AssertionError: expected 1 to equal 0
            at Context.<anonymous> (webpack:///test/TypeaheadSpec.js:148:37 <- tests.webpack.js:19148:50)
............................
Chrome 54.0.2840 (Mac OS X 10.12.2): Executed 60 of 60 (6 FAILED) (1.071 secs / 0.874 secs)
npm ERR! Test failed.  See above for more details.
```

**Comment by ericgio** [comment]
I found 2 issues related to this:

1. For newer versions of React, `ReactTestUtils` appears to no longer be available in `react/lib`. Updated to use `react-addons-test-utils` module.
2. For the issue you're seeing, it looks like lodash's `pick` module is no longer working correctly, so [these lines](https://github.com/ericgio/react-bootstrap-typeahead/blob/master/src/TypeaheadMenu.react.js#L46-L53) don't pass the correct props on to the `Menu` component.

Installing the `lodash.pick` module directly appears to fix the issue. Related, also found that `omit` didn't work properly either and was causing a test warning.
